postgresql/shell-scripts/unix/create-db-changelog.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# Shell script to create a changelog extension of the 3DCityDB
+# on PostgreSQL/PostGIS
+
+# Get the current directory path of this script file
+CURRENT_DIR="$( cd "$( dirname "$0" )" && pwd )"
+if [ $# -ne 0 ]; then
+  source "$1"
+else
+  if [ -f connection-details.sh ]; then
+	  source connection-details.sh
+  else
+	  source "$CURRENT_DIR/connection-details.sh"
+  fi
+fi
+
+# Add PGBIN to PATH
+export PATH="$PGBIN:$PATH"
+
+# Welcome message
+echo ' _______   ___ _ _        ___  ___ '
+echo '|__ /   \ / __(_) |_ _  _|   \| _ )'
+echo ' |_ \ |) | (__| |  _| || | |) | _ \'
+echo '|___/___/ \___|_|\__|\_, |___/|___/'
+echo '                     |__/          '
+echo
+echo '3D City Database - The Open Source CityGML Database'
+echo
+echo '######################################################################################'
+echo
+echo 'Welcome to the 3DCityDB Setup Script. This script will guide you through the process'
+echo 'of setting up a 3DCityDB instance. Please follow the instructions of the script.'
+echo 'Enter the required parameters when prompted and press ENTER to confirm.'
+echo 'Just press ENTER to use the default values.'
+echo
+echo 'Documentation and help:'
+echo '   3DCityDB website:    https://www.3dcitydb.org'
+echo '   3DCityDB on GitHub:  https://github.com/3dcitydb'
+echo
+echo 'Having problems or need support?'
+echo '   Please file an issue here:'
+echo '   https://github.com/3dcitydb/3dcitydb/issues'
+echo
+echo '######################################################################################'
+
+# List the existing 3DCityDB schemas -------------------------------------
+echo
+echo "Reading 3DCityDB schemas from \"$PGUSER@$PGHOST:$PGPORT/$CITYDB\" ..."
+psql -d "$CITYDB" -f "$CURRENT_DIR/../../sql-scripts/util/list-schemas.sql"
```

Comment:
The `list-schemas.sql` script returns all 3DCityDB schemas without checking whether or not the changelog extension is installed. I think we should use an adapted version here (like we use an adapted `list-schemas-with-access-grant.sql` for granting/revoking access).

When installing the changelog extension, it would be nice to just show those 3DCityDB instances for which the changelog extension is not installed already. And the other way round for dropping it. 

Reason: currently, the create-db-changelog script runs into errors when the changelog is already installed in the schema. Likewise, drop-db-changelog shows error if the schema does not contain the changelog extension.

Reply:
Fixed with [f85d574](https://github.com/3dcitydb/3dcitydb/pull/226/commits/f85d57478a4343404e944d8b1d2a5231ff47bfe4)